get_deps.sh

```diff
@@ -112,3 +113,19 @@ export USE_OPENMP=$USE_OPENMP
 EOF
     source deps/env.sh
 fi
+
+if [[ $USE_OMPSS -eq 1 ]]; then
+    export OMPSS_DL_DIR="$PWD"/deps/ompss
+    cat >>deps/env.sh <<EOF
+export USE_OMPSS=$USE_OMPSS
+export OMPSS_DL_DIR=$OMPSS_DL_DIR
+export NANOS_SRC_DIR=$OMPSS_DL_DIR/nanox-0.14.1
+export NANOS_BUILD=$OMPSS_DL_DIR/nanos
+export MERCURIUM_SRC_DIR=$OMPSS_DL_DIR/mcxx-2.1.0
+export MERCURIUM_BUILD=$OMPSS_DL_DIR/mercurium
```

Comment:
The two `_BUILD` variables seem to be the installation directories? Perhaps call them `_PREFIX`?